protocol/driver/driver.go

```diff
@@ -148,12 +148,15 @@ func (c *parseNativeASTCommand) Execute(args []string) error {
 
 	nc, err := native.ExecClient(c.NativeBin)
 	if err != nil {
-		return fmt.Errorf("error executing native client: %s", c)
+		return fmt.Errorf("error executing native client: %s", err.Error())
 	}
 
 	defer func() { _ = nc.Close() }()
 
-	resp, err := nc.ParseNativeAST(&native.ParseASTRequest{Content: string(b)})
+	resp, err := nc.ParseNativeAST(&native.ParseASTRequest{
```

Comment:
It'd be better to separate purely stylistic/unrelated changes in its own commit/PR in the future, since these changes are unrelated to the addition of the encoding field, and it makes it easier to distinguish between them.